common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -158,7 +160,17 @@ public BrokerMsalController(@NonNull final Context applicationContext,
         this(applicationContext,
                 components,
                 activeBrokerPackageName,
-                OneAuthSharedFunctions.getIpcStrategies(applicationContext, activeBrokerPackageName));
+                null);
+    }
+
+    /** Should only be invoked in Background thread, given that getIpcStrategies could be a long running operation. */
+    @WorkerThread
+    private BrokerOperationExecutor getBrokerOperationExecutor(){
+        return new BrokerOperationExecutor(
```

Comment:
Should we create a new instance everytime, or store and reuse?

Reply:
Yeah we can do that